docs/config-example.md

```diff
@@ -42,7 +42,16 @@ spec:
     # Specify additional packages to be installed. The ISO file which is contained in the artifact is required.
     debs: 
       - nfs-common
-    #preInstall:  # Specify custom init shell scripts for each nodes, and execute according to the list order.
+    # initials:
```

Comment:
I think `preInstall` may be more intuitive than `initials`, kk also supports configuring `postInstall`, and some users are already used `preInstall` in their configuration files.

Reply:
Yes, `preInstall` looks like more reasonable. But it runs after the checking. For instance: `socat` is required, but I cannot install `socat` during `preInstall`. Because it will fail before start `preInstall`.

Reply:
Any  suggestions?

Reply:
kk has a command `kk init os -f config.yaml` to install dependency packages， such as: `socat`、`conntrack` .

The original idea of `preInstall` is to support users to do some custom operations before installing kubernetes cluster. such as partitioning disks on nodes, mounting disk to the data directory of docker, and so on.

So I think we can use `kk init os -f config.yaml` to install the necessary dependency packages, and other additional operations can be configured through `preInstall`.

Reply:
If it is an offline environment, kk also supports specifying `--with-packages` to install operating system dependencies using `rpms` or `debs` in offline packages.

https://github.com/kubesphere/kubekey/blob/master/docs/manifest_and_artifact.md
https://github.com/kubesphere/kubekey/blob/master/docs/manifest-example.md

Reply:
I still believe the `preInstall` is not a real `pre` action. It's too late. An easy way to fix that is moving `preInstall` to the first (or second) stage. I just want to do something before any verifies.

Reply:
I got it. I think it‘s ok to advance the execution order of `preInstall`.

So, I think we should directly advance the execution of `preInstall`, and don't need to add this parameter `Initials`, about you?

Reply:
I'm going to do it if you agree with the solution to move `preInstall` after `GreetingsModule`.

Reply:
Yes, I think it is reasonable to do so.  Thanks !

Reply:
Done!